### PR TITLE
created dimension-prefix for subgroup rows. Expanded ELAS -> English …

### PR DESCRIPTION
**Pull request by dsnyder-fairway**
…Language Acquisition Status on Advanced FIlters as there is enough room to provide clarity

### CODE REVIEW

File: webapp/src/main/webapp/src/assets/i18n/en.json [hunk 1]
```diff
@@ -423,6 +423,18 @@
       "Section504": "504",
       "StudentEnrolledGrade": "Student Enrolled Grade"
     },
+    "dimension-prefix": {
+      "EconomicDisadvantage": "Economic Disadvantage",
+      "Ethnicity": "Race / Ethnicity",
+      "Gender": "Gender",
+      "IEP": "IEP",
+      "LEP": "English Learners",
+      "ELAS": "ELAS",
```
Comment (dsnyder-fairway) [inline webapp/src/main/webapp/src/assets/i18n/en.json]: Only thing different from `common.dimension`

File: webapp/src/main/webapp/src/assets/i18n/en.json [hunk 2]
```diff
@@ -57,7 +57,7 @@
       "assessment-grades-info": "The grade of the assessment that was administered",
       "assessment-grades-empty-error": "Please select an assessment grade.",
       "assessment-type-label": "Assessment Type",
-      "elas-label": "ELAS",
+      "elas-label": "English Language Acquisition Status",
```
Comment (dsnyder-fairway) [inline webapp/src/main/webapp/src/assets/i18n/en.json]: expanding this provides a little more clarity on what `English Language Acquisition Status` under the subgroups is actually doing

File: webapp/src/main/webapp/src/assets/i18n/en.json (hunk 1 above)
Comment (malaffoon) [reply webapp/src/main/webapp/src/assets/i18n/en.json]: I'd suggest we shorten the "Ethnicity" prefix. I'd vote for just "Ethnicity".

Comment (malaffoon) [reply webapp/src/main/webapp/src/assets/i18n/en.json]: Perhaps the StudentEnrolledGrade could be shortened as well, to just "Enrolled Grade".

File: webapp/src/main/webapp/src/assets/i18n/en.json [hunk 3]
```diff
@@ -423,6 +423,18 @@
...
+      "Ethnicity": "Ethnicity",
+      "Gender": "Gender",
+      "IEP": "IEP",
+      "LEP": "English Learners",
+      "ELAS": "ELAS",
+      "MigrantStatus": "Migrant Status",
+      "Overall": "Overall",
+      "Section504": "504",
+      "StudentEnrolledGrade": "Enrolled Grade"
+    },
```
Comment (esotrope) [inline webapp/src/main/webapp/src/assets/i18n/en.json]: so what is the semantic diff between "dimension" and "dimension-prefix" ? "prefix" doesn't tell me much

Comment (dsnyder-fairway) [reply webapp/src/main/webapp/src/assets/i18n/en.json]: I agree. I'm open to suggestions. "subgroup-row-prefix" or "dimension-row-prefix" would be more appropriate. Thoughts?

Comment (esotrope) [reply webapp/src/main/webapp/src/assets/i18n/en.json]: Hm. Is the intentio just to shorten it? Maybe we just call it `dimension-short-name` like the other stuff

Comment (esotrope) [reply webapp/src/main/webapp/src/assets/i18n/en.json]: I'm wondering if our short displays shouldn't group too like:
Gender: [Male] [Female] Ethnicity: [White] [Asian]

Comment (dsnyder-fairway) [reply webapp/src/main/webapp/src/assets/i18n/en.json]: I don't understand what you mean. Here's an example of the strings in use
screen shot 2018-04-05 at 11 37 15 am

Are you referencing the Report Review section?

**Pull request merged**